src/java.desktop/share/classes/javax/swing/text/html/parser/Parser.java

```diff
@@ -25,15 +25,15 @@
 
 package javax.swing.text.html.parser;
 
+import java.io.CharArrayReader;
```

Comment:
We can revert all changes to this file now.

Reply:
Thanks for catching that. Reverted